index.html

```diff
@@ -0,0 +1,273 @@
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <div class="header-container">
+    <div class="logo">
+      <a href="https://github.com/OLIPLICHE/portfolio/pull/1">My Logo</a>
+    </div>
+    <nav class="navbar">
+      <ul class="d-navbar">
+        <li><a class="a-navbar" href="#Portfolio">Portfolio</a></li>
+        <li><a class="a-navbar" href="#About">About</a></li>
+        <li><a class="a-navbar" href="#Contact">Contact</a></li>
+      </ul>
+      <div class="humberger" >
+        <i class="fa fa-bars"></i>
+      </div>
+    </nav>
+  </div>
+  </header>
+  <main id="main-container">
+    <section class="headline">
+      <div class="headline-main-container">
+        <h1 class="welcome">I am Paka <br> Glad to see <br> you!</h1>
+        <p class="introduction">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <div class="link-up">
+        <h2>LET'S CONNECT</h2>
+        <div class="social-list">
+          <ul>
+            <li>
+              <a href="#"><img alt="github" src="image/github.png"/></a>
+            </li>
+            <li>
+              <a href="#"><img alt="twitter" src="image/twitter.png"/></a>
+            </li>
+            <li>
+              <a href="#"><img src="image/Linkedin_icon.png" alt="linkedin" /></a>
+            </li>
+            <li>
+              <a href="#"><img alt="medium" src="image/medium.png"/></a>
+            </li>
+            <li>
+              <a href="#"><img alt="angellist" src="image/angellist.png"/></a>
+            </li>   
+           </ul>  
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <section id="workers">
+    <ul class="boxes">
+      <li class="box">
+        <div class="Tonic">
+          <img src="image/Snapshoot_Portfolio.svg" class="img" alt="tonic">
+        </div >
+        <div class="Tonic-title">
+          <h4>Tonic</h4>
+          <div class="menu">
+            <ul class="Tonic-menu">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="introduction">A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </p>
+          <div class="language">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
```

Comment:
you need to style the languages for each project as shown below
### Original design
![right](https://user-images.githubusercontent.com/58376020/126038810-5a879fb1-f067-4d25-adbf-92b0c6dbaf7b.png)
###Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126038830-eeae710b-3b53-431d-b34d-ea4779a166a6.png)